Dispatcher/Repository/ArticleRepository.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+protocol ArticleRepository {
```

Comment:
this is not an actual repository but an interface for one 
it should be 
ArticleRepositoryProtocol or ArticleRepositoryProt